sympy/polys/polytools.py

```diff
@@ -3282,8 +3282,8 @@ def intervals(f, all=False, eps=None, inf=None, sup=None, fast=False, sqf=False)
 
         For real roots the Vincent-Akritas-Strzebonski (VAS) continued fractions method is used.
 
-        References:
-        ===========
+        References
+        ==========
            1. Alkiviadis G. Akritas and Adam W. Strzebonski: A Comparative Study of Two Real Root
```

Comment:
Something wrong here. Maybe indentation.

Reply:
Seems like Sphinx doesn't like the "1." at the beginning of the reference here. Corrected it to use the correct syntax for a footnote like I've seen in other parts of the codebase.